jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1942,12 +1924,12 @@ typedef struct
  * If ecma_prop_hashmap_alloc_state other other than this value, it is
  * disabled.
  */
-#define ECMA_PROP_HASHMAP_ALLOC_ON 0
+#define JERRY_PROP_HASHMAP_ALLOC_ON 0
```

Comment:
These should not be changed, only those which are defined in jerryscript-types.h